src/libraries/System.IO.Compression/src/System/IO/Compression/CompressionLevel.cs

```diff
@@ -3,27 +3,19 @@
 
 namespace System.IO.Compression
 {
-    /// <summary>
-    /// Specifies values that indicate whether a compression operation emphasizes speed or compression size.
-    /// </summary>
-
-    // This is an abstract concept and NOT the ZLib compression level.
-    // There may or may not be any correspondence with the a possible implementation-specific level-parameter of the deflater.
+    /// <summary>Specifies values that indicate whether a compression operation emphasizes speed or compression size.</summary>
+    /// <remarks><format type="text/markdown"><![CDATA[
+    /// [!INCLUDE[remarks](~/includes/remarks/System.IO.Compression/CompressionLevel/CompressionLevel.md)]
```

Comment:
This link can't be followed from runtime's source... is there a value on porting it? Perhaps you could ignore it and keep the source of truth on docs.

Reply:
My understanding [from the docs](https://github.com/carlossanlop/DocsPortingTool/blob/main/BackportInstructions.md#user-content-instructions:~:text=add%20an%20INCLUDE%20markdown%20element%20that%20points%20to%20the%20newly%20created%20file) is that we do want to port these links over.